### PR TITLE
Eslint: use `hermes-eslint` parser (experimental)

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

**Comment by mrtnzlml** [comment]
Known issue no. 1: how to deal with Flow enums?

Screen Shot 2022-06-18 at 12 34 49

**Comment by mrtnzlml** [comment]
Known issue no. 2: some parsing errors

Screen Shot 2022-06-18 at 12 37 19

File: src/eslint-config-adeira/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+TODO: probably cut a minor release first (as this is potentially a breaking change)
```
Comment (mrtnzlml) [inline src/eslint-config-adeira/CHANGELOG.md:3]: TODO: document the changes (turned off rules, why)

**Comment by mrtnzlml** [comment]
Relevant discussion: https://github.com/flow-typed/eslint-plugin-ft-flow/issues/33

**Comment by mrtnzlml** [comment]
Hey @bradzacher! 👋 I noticed that parsing for `import.meta` has been implemented (https://github.com/facebook/hermes/commit/dd4c01d6a08f13ce6217b179891dc6f138f72739). Do you think you could release a new version of `hermes-eslint` with this change so I can land this PR? 🙂

**Comment by bradzacher** [comment]
@mrtnzlml yeah sorry there was another change I was waiting to land before doing the release which landed yesterday.

I'll do the bump and release on Monday morning!

**Comment by mrtnzlml** [comment]
@bradzacher, I've just landed this PR. Thanks for your help! 👍

**Pull request merged**